packages/template-ui/channel-overrides/cas-es/styles.scss

```diff
@@ -0,0 +1,2 @@
+$primary: #166572;
+$secondary: #87D036;
```

Comment:
Ah, yes I forgot to mention this! Changing the colors like this would make CAS (es) different than CAS, which would be inconsistent. Same as the options mentioned here: https://github.com/endlessm/kolibri-explore-plugin/issues/635#issuecomment-1602519791

I want to wait for Joana to tell us if the change is intentional, if it's an improvement for both channels or not.

Reply:
I have updated the comment asking Joana to clarify.

Reply:
My impression from the last team meeting was that they wanted a different visual identity between CAS-en and CAS-es because they have different content, it is not a translation of the same content, so my guess is that the change is intentional and only applies to CAS-es. But let's wait to hear back from Joana.

Reply:
Yes, as the Spanish channel is not just a translation those channels should look slightly different.

Reply:
https://github.com/endlessm/kolibri-explore-plugin/issues/635#issuecomment-1602775360